x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="FreeType-based font drawing library for X"
+DESCRIPTION="Xft provides a client-side font API for X applications, \
+making the FreeType font rasterizer available to X clients. Fontconfig \
+is used for font specification resolution. Where available, the RENDER \
+extension handles glyph drawing; otherwise, the core X protocol is used."
```

Comment:
```diff
-DESCRIPTION="Xft provides a client-side font API for X applications, \
+DESCRIPTION="Xft provides a client-side font API for X applications, making \
-making the FreeType font rasterizer available to X clients. Fontconfig \
+the FreeType font rasterizer available to X clients. Fontconfig is used for \
-is used for font specification resolution. Where available, the RENDER \
+font specification resolution. Where available, the RENDER extension handles \
-extension handles glyph drawing; otherwise, the core X protocol is used."
+glyph drawing; otherwise, the core X protocol is used."
```
This is not strictly required, but it would be nice to fill the lines as much as possible (i.e. to have lines <= 80 chars, including the `\`). Thanks!

Reply:
Fixed.